docs/rules/no-disallowed-properties.md

```diff
@@ -0,0 +1,30 @@
+# Ban properties
+
+Rule `no-disallowed-properties` will disallow usage of certain properties
```

Comment:
If we're going for consistency we could update this to;

Rule `no-disallowed-properties` will warn against the use of certain properties. 
(Or maybe 'user-specified properties'??).